pkg/dependency/parser/sbt/lockfile/parse.go

```diff
@@ -0,0 +1,77 @@
+package lockfile
+
+import (
+	"io"
+
+	"github.com/liamg/jfather"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/dependency"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
+	xio "github.com/aquasecurity/trivy/pkg/x/io"
+)
+
+// lockfile format defined at: https://stringbean.github.io/sbt-dependency-lock/file-formats/version-1.html
+type sbtLockfile struct {
+	Version        int                     `json:"lockVersion"`
+	Timestamp      string                  `json:"timestamp"`
+	Configurations []string                `json:"configurations"`
+	Dependencies   []sbtLockfileDependency `json:"dependencies"`
+}
+
+type sbtLockfileDependency struct {
+	Organization   string                `json:"org"`
+	Name           string                `json:"name"`
+	Version        string                `json:"version"`
+	Artifacts      []sbtLockfileArtifact `json:"artifacts"`
+	Configurations []string              `json:"configurations"`
+	StartLine      int
+	EndLine        int
+}
+
+type sbtLockfileArtifact struct {
+	Name string `json:"name"`
+	Hash string `json:"hash"`
+}
+
+type Parser struct{}
+
+func NewParser() *Parser {
+	return &Parser{}
+}
+
+func (Parser) Parse(r xio.ReadSeekerAt) ([]ftypes.Package, []ftypes.Dependency, error) {
+	var lockfile sbtLockfile
+	input, err := io.ReadAll(r)
+
+	if err != nil {
+		return nil, nil, xerrors.Errorf("failed to read sbt lockfile: %w", err)
+	}
+	if err := jfather.Unmarshal(input, &lockfile); err != nil {
+		return nil, nil, xerrors.Errorf("JSON decoding failed: %w", err)
+	}
+
+	var libraries []ftypes.Package
+
+	for _, dep := range lockfile.Dependencies {
+		libraries = append(libraries, ftypes.Package{
+			ID:        dependency.ID(ftypes.Sbt, dep.Organization+":"+dep.Name, dep.Version),
+			Name:      dep.Organization + ":" + dep.Name,
+			Version:   dep.Version,
+			Locations: []ftypes.Location{{StartLine: dep.StartLine, EndLine: dep.EndLine}},
+		})
+	}
+
+	return libraries, nil, nil
```

Comment:
```suggestion
	sort.Sort(libraries)
	return libraries, nil, nil
```
- use `var libraries ftypes.Packages` for this.
- you can remove this after that: https://github.com/aquasecurity/trivy/blob/1890227545bb7bb1f2cd31a85285a6465ba11a72/pkg/dependency/parser/sbt/lockfile/parse_test.go#L82-L90